.github/workflows/wheels.yaml

```diff
@@ -0,0 +1,68 @@
+name: Build Wheels
+on: push
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}-${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, macos-13]
+        arch: [x86_64]
+        include:
+          - os: macos-13
+            arch: arm64
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.16.1
+        with:
+          package-dir: py-bindings
+        env:
+          CIBW_ARCHS_MACOS: ${{ matrix.arch }}
+          OMPL_BUILD_ARCH: ${{ matrix.arch }}
+          CIBW_BUILD: cp310-macosx_{x86_64,arm64} cp310-manylinux_x86_64
+          # NOTE: Many combinations of OS, arch, and Python version can be built
+          # depending on your patience. For example:
+          # CIBW_BUILD: cp3{10,11,12}-macosx_{x86_64,arm64} cp3{9,10,11,12}-manylinux_x86_64
```

Comment:
Which combinations should we build for in CI? It's only a matter of patience and not exceeding Github's CI job timeout.

Maybe:
- Linux cp3{10,11,12} to support Ubuntu 22.04+
- Mac OS cp12 to match the latest Python version in Brew

Reply:
Maybe also Linux cp39? Some Noetic environments still use Python 3.9 iirc.

Reply:
cp38 and I believe ISAAC Sim is tied to cp37 may be worthwhile targets... but that may be pushing the amount of time we have on the CI job.

Reply:
cp37-cp312 might be possible. I think the default Github CI limit is 6hrs. I'll hold off until we're done with reviews and then test it out in CI to avoid making the builds take forever during the review process.

Reply:
Yep, I enabled for: `cp3{10,11,12}-macosx_{x86_64,arm64} cp3{7,8,9,10,11,12}-manylinux_x86_64`. Seems to only take about 2hrs on CI, so plenty of time to spare.